src/interfaces/filing-interfaces/filing-interfaces.ts

```diff
@@ -184,9 +184,9 @@ export interface RestorationFilingIF {
   }
   // *** TODO: update according to schema
   restoration: {
-    date: string
+    date: string // today, as YYYY-MM-DD
     type: RestorationTypes
-    expiry: string
+    expiry?: string // YYYY-MM-DD
```

Comment:
`expiry` is only saved in the JSON object for a Limited Restoration.

It will need to be computed as "today + num_days" (ie, whatever future option the user has selected).

I think we will also need to save the future option that the user has selected, so it can resumed later, and so we can compute the expiry date later.

Note that the expiry date is displayed on the Review and Confirm page.